src/static/js/header_additional.js

```diff
@@ -278,6 +278,22 @@ additional = {
         return valid;
     },
 
+    reset: function(){
+        //animal
+        $(".asm-animalchooser").animalchooser("clear");
+        if ($("#add_9.additional option[selected=selected]")[0])
```

Comment:
what element is add_9 ? Can't see any changes to render method to add this. Are you referring to an additional field that might not be present in other databases?